src/figuresClasses.ts

```diff
@@ -1,19 +1,99 @@
-export interface Figure {
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'green' | 'red' | 'blue';
 
+export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea(): number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape: Shape;
```

Comment:
We don't need to set this value in the constructor
```suggestion
  shape: Shape = 'triangle';
```